src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/ShapePlacementStrategy/ShapePlacementParsingStrategy.cs

```diff
@@ -70,6 +70,8 @@ private void ProcessFeatureDescriptor(ShapeTableBuilder builder, IFeatureInfo fe
 
         private void ProcessPlacementFile(ShapeTableBuilder builder, IFeatureInfo featureDescriptor, PlacementFile placementFile)
         {
+            if (placementFile == null) return;
```

Comment:
I think you should figure out why `placementFile` is null instead. Check out why `serializer.Deserialize<PlacementFile>(jtr);` is returning null.
You may want to add curly brackets. I think that is becoming the new standard in OC.

Reply:
I dislike checking the params in `private` functions. As @CrestApps said, try to figure what leading the placement file to be `null`

Reply:
null should be checked line # 63 not here